src/vault.ts

```diff
@@ -7,28 +7,19 @@ interface IVaultOptions {
 }
 
 export default class Vault<StorableItem extends Dictionary> {
```

Comment:
Does this still need to extend dictionary?  I removed it locally, ran the tests and the tests still pass.